app/app.css

```diff
@@ -1,15 +1,45 @@
+@import url('//cdn.jsdelivr.net/gh/neodgm/neodgm-webfont@1.600/neodgm/style.css');
+@import url("https://font.elice.io/css?family=Elice+Digital+Baeum");
 @import "tailwindcss";
 
+
 @theme {
   --font-sans: "Inter", ui-sans-serif, system-ui, sans-serif,
     "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 }
 
-html,
-body {
-  /* @apply bg-white dark:bg-gray-950;
+.font-apple-gothic {
+    font-family: 'Apple SD Gothic Neo', sans-serif;
+  }
+  
```

Comment:
https://tailwindcss.com/docs/font-family#customizing-your-theme
<img width="707" alt="image" src="https://github.com/user-attachments/assets/247fb6b2-356a-4143-8329-018ab6f3002a" />

공식 문서 보면, 폰트들을 클래스 명으로 쓸 수 있게 theme으로 다 추가를 할 수 있는 거 같아요!
아래처럼 모든 폰트들에 대해 theme으로 추가하는 게 어떤가요?

```suggestion
@theme {
  --font-sans: "Inter", ui-sans-serif, system-ui, sans-serif,
      "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
  --font-apple-gothic: 'Apple SD Gothic Neo', sans-serif;
  / * 아래 폰트들도 추가 */
}
```

Reply:
오 +-+ !! 몰랐네요 반영해두겠습니다 감사합니다 ㅎㅎ

Reply:
https://github.com/mash-up-kr/GGLK-Web/pull/8/commits/51d7363c26face0ee131780ff185baf79a434822

cc. @Grit03